src/mlpack/bindings/R/mlpack/src/Makevars.win

```diff
@@ -0,0 +1,3 @@
+PKG_CXXFLAGS = -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=false -I. $(SHLIB_OPENMP_CXXFLAGS)
```

Comment:
Last think that I have to point out is -- After the [discussion](https://github.com/Yashwants19/RcppMLPACK/pull/11#discussion_r456810544), I have removed `ftrack-macro-expansion=0` from this file, which was resolving memory issue on Windows for CI.
Note : Locally on my system it is working fine without this flag.

Reply:
For the safe side should I check the package using `devtools::check_win_devel()`, or should we continue without this flag?

Reply:
It seems like `-ftrack-macro-expansion` is only relevant for printing error messages.  I'm surprised that `0` causes a failure, since that just disables tracking.  Anyway, I have no idea if this will matter for CRAN; @coatless or @eddelbuettel will have a better idea.

Reply:
I have tried `devtools::check_win_devel()` for the package:
Without `ftrack-macro-expansion=0` flag: I have got the same memory error.
https://win-builder.r-project.org/k0ha3e9083DC/

With `ftrack-macro-expansion=0` flag: it worked
https://win-builder.r-project.org/fd7zTJ71ITh4

Now the final call is of @coatless and @eddelbuettel. If we are okay without this flag, we are ready for merge, else I will update the `Makevars.win` file and add this flag, then we will be ready for merge.

NOTE: It was confirmed in one of [comment](https://github.com/Yashwants19/RcppMLPACK/pull/11#discussion_r456811337) by @coatless that CRAN doesn't hate this flag.

Reply:
Sounds good to me---since the build is passing now, I think we can go with it as you have it now, and if there's an issue with CRAN later, we can resolve it then. :+1:

Reply:
If it's required to ensure the build passes, lets leave it in. We're going to probably need a few back'n'forths with CRAN to get the package listed.

Reply:
Sounds awesome, Thank you for all the clarification. :)